index.html

```diff
@@ -104,13 +102,16 @@ <h2 class="section-title">My Work</h2>
             <p class="lead">Check Out Some Of My Projects</p>
             <div class="items">
               <div class="item">
-                <div class="item-image">
-                  <img src="img/items/item1.png" alt="" />
+                <div class="terrarium">
+                  <img src="img/items/terrarium.png" class="terrarium" alt="" />
```

Comment:
I would keep the `item-image` class on the `<div>`. This is the class that adds the zoom animation on the image when you hover over it.

You can set the height of this image to 100% to make it fill the div or you can use a new image that has an aspect ratio of 4:3. The image size should be something like 400px by 300px. That should allow it to naturally fill, without distorting the image

Reply:
Also, consider using a smaller image here. Larger images have larger file sizes and takes longer to load. We want to ensure that your site can load fast, even for people with slower internet connections. The rule of thumb is to use an image size that matches how large it will be shown on the site. So if the image is set to 200px by 200px, we want to use an image that is 200px by 200px.

The current image dimension is 2750px by 1488px and is 1.4 MB, but you can scale it down to 924px by 500px.

Reply:
Thanks for the suggestions, Kevin! I'll implement the changes.